src/react/account/Accounts.jsx

```diff
@@ -20,6 +21,11 @@ const Accounts = () => {
     const accounts = useSelector((state: AppState) => state.configuration.latest.users);
     const accountList = useMemo(() => sortByDate(accounts, false), [accounts]);
     const accountIndex = useMemo(() => accountList.findIndex(a => a.userName === accountNameParam), [accountList, accountNameParam]);
+    const account = accountList[accountIndex];
+
+    if (account) {
+        dispatch(selectAccountID(account.id));
```

Comment:
`selectAccountID` is a side effects operation. We might want to use React `useEffect` method?
Also, I do not think we want to "switch account" (reloading clients and re-fetching items) every time we select a new account.
Instead, what we can do for `deleteAccountAccessKey` and `listAccountAccessKey` actions method (in actions/account.js):
- retrieve temporary keys with STS.assumeRoleWithWebIdentity (generate role with the right account name)
- instantiate new IAM client  with the temporary keys.
- call the IAM respective method.

Reply:
done